### PR TITLE
Update start.md

### LINKED ISSUE #408: Refining the tutorial documentation
Read and try out the examples and tutorials in "[Getting Started](https://autokeras.com/tutorial/overview/)".
If you find anything unclear or incorrect, please change it in [start.md](https://github.com/keras-team/autokeras/tree/master/docs/templates/tutorial).

External contributors are welcome for solving this issue.

- [ ] Resume training.
- [ ] ImageRegressor
- [ ] TextRegressor
- [ ] StructuredDataRegressor
- [ ] customize metric and loss

### PR DESCRIPTION
**Pull request by francotheengineer**
Added an example of how to make a CSV for use with the function `load_image_dataset` to load images.

This pull request resolves #408.

The pull request is ready to be merged.

The details of the pull request:

Added an example to generate a CSV file containing lists of images and their respective labels.

### CODE REVIEW

**Comment by jhfjhfj1** [comment]
Thank you very much for the contribution! @francotheengineer
